internal/cryptoutil/cryptoutil.go

```diff
@@ -115,6 +116,37 @@ func LoadCertificate(kmsURI, certPath string) ([]*x509.Certificate, error) {
 	return cert, nil
 }
 
+// LoadTLSCertificate returns a [tls.Certificate] from a certificate fine and a
+// key in a file or in a KMS.
+func LoadTLSCertificate(certFile, keyName string, opts ...pemutil.Options) (tls.Certificate, error) {
+	bundle, err := pemutil.ReadCertificateBundle(certFile)
+	if err != nil {
+		return tls.Certificate{}, err
+	}
+
+	var signer crypto.Signer
+	if IsKMS(keyName) {
+		if signer, err = CreateSigner(keyName, keyName, opts...); err != nil {
+			return tls.Certificate{}, err
+		}
+	} else {
+		if signer, err = CreateSigner("", keyName, opts...); err != nil {
+			return tls.Certificate{}, err
+		}
+	}
```

Comment:
I like this. I might be able to use it in #1399 too (when we support creating TSS2 signers from files).